Dimas Agung Yudono

dimasday90 opened the pull request:
initial commit firebase test

dimasday90 committed:
initial commit firebase test

dimasday90 committed:
Saas and index.html updates:
1. adding header for navbar,
2. adding creating board container for kanban boards,
3. using materialize as css framework,
4. working with sass for css and compiled

dimasday90 committed:
masih tinggal google sign in

dimasday90 committed:
modified index.html

dimasday90 committed:
firebase deploy done

dimasday90 commented:
link firebaseapp kanban:
https://kanban-5ff2b.firebaseapp.com
sudah tersambung dengan heroku

kendala:
- google sign in belum berhasil setelah di deploy,
- menu dropdown tidak bekerja setelah login.. namun ketika di refresh, dropdown menu berfungsi dengan baik,
- beberapa fitur dari server belum disambungkan, diantaranya:
  - menambah user ke satu organization,
  - menambah member di dalam organization untuk join langsung ke suatu task,
  - menghapus member dari task di dalam organization

dhimashary commented on src/components/KanbanBoard.vue:
```diff
@@ -0,0 +1,209 @@
+        <div class="row">
+            <div class="col s12 m6 l3">
+                <div id="backlog-column" class="t col s12">
+                    <div class="card-title center-align activator">
```
BackLog, Product, Development, Done bagusnya dipecah lagi menjadi satu komponen jadi disini kita hanya perlu <KanbanContainer :type="BackLog" :task=""/> <KanbanContainer :type="Product" :task=""/> dst

dhimashary commented:
- apakah google client id sudah sesuai dengan link kanban-5ff2b.firebaseapp.com ketika megenerate client id? 
- firebaserc di ignore
- tulisan helper text di register tolong dihapus ya
- ketika add organization / add task klik placeholdernya gak bisa (kayak disabled), musti klik persis di garisnya
- server dan client juga harus handle validasi tasknya, disini kalo ga isi title dan body masih bisa masuk
- UI ketika update gak keliatan kalo udah keisi formnya padahaal udah keisi

dimasday90 committed:
delete .firebaserc and modified

dimasday90 committed:
Revised:
delete helper text in register form,
replace v-if and v-else to v-show in Header.vue,
adding created in every form view
undone auto fill handling when direct to update form

dimasday90 commented:
Resolved some bugs:
deleted .firebaserc cache from git,
deleted helper text from register form,
added created in every form vue file,

Undone fixing auto fill in update form and other UI form
